package.json

```diff
@@ -35,7 +35,9 @@
     "rollup-plugin-json": "2.1.0",
     "rollup-plugin-node-builtins": "2.0.0",
     "rollup-plugin-node-globals": "1.1.0",
-    "rollup-plugin-node-resolve": "2.0.0"
+    "rollup-plugin-node-resolve": "2.0.0",
+    "typescript": "2.2.1",
```

Comment:
this will need to be a "dependency" if you want people to use it in prod. But we can leave it here right now as we are experimenting with it.